src/components/tabs/Tabs.vue

```diff
@@ -31,7 +31,30 @@
         :class="['tabs__content', { 'is-active': activeTabId === tab.id }]"
         :aria-labelledby="tab.id + '-tab'"
       >
-        {{ tab.content }}
+        <dl>
+          <div>
+            <dt>Interest free period</dt>
+            <dd>{{ tab.content.interestFree }}</dd>
+          </div>
+          <div>
+            <dt>Purchase amount</dt>
+            <dd>{{ tab.content.purchaseAmount }}</dd>
+          </div>
+          <div>
+            <dt>Establishment fee</dt>
+            <dd>{{ tab.content.estFee }}</dd>
+          </div>
+          <div>
+            <dt>Indicative minimum monthly repayments*</dt>
+            <dd>{{ tab.content.minimumMonthlyPayment }}</dd>
+          </div>
+          <div>
+            <dt>
+              Indicative repayment to pay before Interest free period expires ^
+            </dt>
+            <dd>{{ tab.content.repayBeforeInterest }}</dd>
+          </div>
```

Comment:
This should not be hardcoded in the tabs. Here you're coupling the list content to the tabs themselves. 
Consider if you need to create one or more components just for datalists, and if you can pass those components through to a named `<slot></slot>` in the Tabs